internal/kafka/internal/config/kafka_supported_instance_types.go

```diff
@@ -128,3 +128,21 @@ func NewKafkaSupportedInstanceTypesConfig() *KafkaSupportedInstanceTypesConfig {
 		ConfigurationFile: "config/kafka-instance-types-configuration.yaml",
 	}
 }
+
+func (s *SupportedKafkaInstanceTypesConfig) GetProfileByID(instanceType string) (KafkaInstanceType, error) {
```

Comment:
Due to the recent change in the namings, I believe this should now be `GetInstanceTypeByID` or `GetInstanceTypeConfigByID`